content/articles/wavelet-transform-based-denoising-of-images-using-matlab/index.md

```diff
@@ -0,0 +1,141 @@
+have been written and some discussed but it remains to be a challenge. We will discuss the currently available method, that is, wavelet-based 
+denoising of images, which is the currently efficient method. Signal transmission over a distance may be contaminated with noise. The objective 
+here is to remove the noise from the image using the wavelet technique. 
+The signal acquires the noise through the additive method and it has the form below.
+f'(x,y) = f(x,y) + n(x,y) where f'(x,y) is the contaminated signal, f(x,y) is the original signal and n(x,y) is the noise signal. The basic assumption of noise signal n(x,y) for the proposed scheme are;
+- Noise is additive
+- Noise is a random signal(white gaussian with zero mean value)
+- Noise has a high frequency.
+The objective here is to remove the noise n(x,y) from the noisy image f'(x,y) using the wavelet technique.
+
+### Prerequisite
+1. [Matlab](https://www.mathworks.com/downloads/) installed.
+2. A proper understanding of [Matlab](https://www.section.io/  engineering-education/getting-started-with-matlab/) language.
+
+### Wavelet-Based Denoising scheme
+The denoising scheme involves passing the signal through decomposition to be decomposed into various wavelet co-efficient 
+using discrete wavelet transform(DWT). The coefficients are then thresholded and then collected together. image thresholding 
+is the separation of foreground and background. It is a method of image segmentation and it isolates grayscale images by 
+converting them to binary images. using the inverse discrete wavelet transform(IDWT) to get the denoised image. Finding threshold 
+labels, we have various methods such as universal threshold, Bayes, SURE, MinMax, e.t.c.
+
+### Matlab's 'WaveletAnalyzer' tool
+This is an in-built tool that can be used for denoising of images in Matlab. To access this tool; 
+- Type `wavelet analyzer` on the command prompt. We get a new window below.
+
+![WaveletAnalyzer tool](/engineering-education/wavelet-transform-based-denoising-of-images/image1.png)
+After accessing the tool, we locate `wavelet 2-D` since the images that we want denoise are 2-D. When you click this, 
+a new window shown below opens up.
+
+![wavelet 2-D](/engineering-education/wavelet-transform-based-denoising-of-images/image2.png)
+On the file folder, click 'load' then 'image' to import the image to be denoised.
+Choose your noisy image to be denoised then import.
+
+![imported image](/engineering-education/wavelet-transform-based-denoising-of-images/image3.png)
+On the top left corner of the window above choose the wavelet(denoising method) e.g bior and the co-efficient e.g 2.2 and the level of decomposition.
+> Note that `Db`, `haar`, `sym`, and `coif` are orthogonal wavelets and have compact support(smaller support).
+`bior`, `rbior` are biorthogonal wavelets and have compact support.
+`mey`, `dmey` are orthogonal wavelets and have non-compact support.
```

Comment:
I'm not sure if these are short hands. This looks unclear. Please brief about this